seacrowd/sea_datasets/emotes_3k/emotes_3k.py

```diff
@@ -0,0 +1,242 @@
+    Tasks.INSTRUCTION_TUNING # FLAN-T5 Training
+] 
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class Emotes3K(datasets.GeneratorBasedBuilder):
+    """
+    Emotes3K consists of one human annotated dataset for the purpose of morality classification and instruction tuning.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="emotes_3k_source",
+            version=SOURCE_VERSION,
+            description="emotes_3k source schema",
+            schema="source",
+            subset_id="emotes_3k",
+        ),
+        SEACrowdConfig(
+            name="emotes_3k_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="emotes_3k SEACrowd schema",
+            schema="seacrowd_text",
+            subset_id="emotes_3k",
+        ),
+        SEACrowdConfig(
+            name="emotes_3k_eng_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="emotes_3k SEACrowd schema",
+            schema="seacrowd_text",
+            subset_id="emotes_3k",
+        ),
+        SEACrowdConfig(
+            name="emotes_3k_tgl_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="emotes_3k SEACrowd schema",
+            schema="seacrowd_text",
+            subset_id="emotes_3k",
+        ),
+        SEACrowdConfig(
+            name="emotes_3k_seacrowd_t2t",
+            version=SEACROWD_VERSION,
+            description="emotes_3k SEACrowd schema",
+            schema="seacrowd_t2t",
+            subset_id="emotes_3k",
+        ),
+        SEACrowdConfig(
+            name="emotes_3k_eng_seacrowd_t2t",
+            version=SEACROWD_VERSION,
+            description="emotes_3k SEACrowd schema",
+            schema="seacrowd_t2t",
+            subset_id="emotes_3k",
+        ),
+        SEACrowdConfig(
+            name="emotes_3k_tgl_seacrowd_t2t",
+            version=SEACROWD_VERSION,
+            description="emotes_3k SEACrowd schema",
+            schema="seacrowd_t2t",
+            subset_id="emotes_3k",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "emotes_3k_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    'entry_id'          : datasets.Value("string"),
+                    'Filipino'          : datasets.Value("string"),
+                    'English'           : datasets.Value("string"),
+                    'Annotation'        : datasets.ClassLabel(names=["Moral", "Immoral"]),
+                    'Explanation'       : datasets.Value("string"),
+                    'Personality Traits': datasets.Value("string"),
+                    'Topic'             : datasets.Value("string"),
+                    'Topic Name'        : datasets.Value("string")
+                }
```

Comment:
Can you run the `make check_file=...` script again? I don't think this is how the formatter usually formats this dictionary.

Reply:
I have an issue with running make check_file sadly :(. 
How was it supposed to be formatted, if you don't mind me asking?

Reply:
@luckysusanto As in previous PRs, I can format this file and commit it before merging